core/consensus/babe/src/lib.rs

```diff
@@ -929,29 +930,13 @@ impl MaybeSpanEpoch {
 	}
 }
 
-/// Extract current epoch data from cache and fallback to querying the runtime
-/// if the cache isn't populated.
-fn epoch<B, C>(client: &C, at: &BlockId<B>) -> Result<MaybeSpanEpoch, ConsensusError> where
+/// Extract current epoch data from the runtime.
+fn epoch<B, C>(client: &C, at: &BlockId<B>) -> Option<MaybeSpanEpoch> where
```

Comment:
I would prefer for such a short function, which is only used once, to be inlined into its call site.

Reply:
it's used in 2 places in this file, as well as in the tests module